src/util/tubemap.js

```diff
@@ -3918,6 +3918,26 @@ function trackDoubleClick() {
   createTubeMap();
 }
 
+// Takes a track and returns a string describing the nodes it passes through
+// In the format of >1>2<3>4, with the intergers being nodeIDs
+function getPathInfo(track) {
+  let result = "";
+  if (!track.sequence) {
+    return result;
+  }
+
+  for (const nodeID of track.sequence) {
+    // Node is approached backwards if "-" is present
+    if (Array.from(nodeID)[0] === "-") {
```

Comment:
This seems like a weird way to see if the string starts with "-", even though it looks like [StackOverflow recommends it](https://stackoverflow.com/a/3427148), since this looks to convert the whole string to an array just to check the first character. I would probably try the `startsWith` method instead.
```suggestion
    if (nodeID.startsWith("-")) {
```